coins.json

```diff
@@ -1290,20 +1290,20 @@
         "symbol": "ADA",
         "decimals": 6,
         "blockchain": "Cardano",
-        "derivationPath": "m/44'/1815'/0'/0/0",
-        "curve": "ed25519Extended",
-        "publicKeyType": "ed25519Extended",
+        "derivationPath": "m/1852'/1815'/0'/0/0", 
```

Comment:
can you remove trailing space? maybe you have to change your editor setting

Reply:
done